sdv/multi_table/base.py

```diff
@@ -205,11 +205,14 @@ def _assign_table_transformers(self, synthesizer, table_name, table_data):
         """Update the ``synthesizer`` to ignore the foreign keys while preprocessing the data."""
         synthesizer.auto_assign_transformers(table_data)
         foreign_key_columns = self._get_all_foreign_keys(table_name)
+        transformers = synthesizer.get_transformers()
         column_name_to_transformers = {
             column_name: None
             for column_name in foreign_key_columns
+            if transformers.get(column_name) is not None
```

Comment:
Does it raise an error if the transformer is already None?

Reply:
No, it was just to avoid updating the transformers if they had already been changed to `None`. Since we're always re-fitting this doesn't matter anymore.